### PR TITLE
Update quickstart docs

### PR DESCRIPTION
**Pull request by m4dcoder**

### CODE REVIEW

File: docs/source/engage.rst [hunk 1]
```diff
@@ -3,4 +3,4 @@
 * IRC: `#stackstorm on irc.freenode.org <http://webchat.freenode.net/?channels=stackstorm>`_
 * Support: support@stackstorm.com
 * Google Group: `stackstorm@googlegroups.com <https://groups.google.com/forum/#!forum/stackstorm>`_
-* **NEW**: Slack community channel: `stackstorm-community.slack.com <https://stackstorm-community.slack.com>`__ (Registr `here <https://stackstorm.typeform.com/to/K76GRP>`__.)
+* **NEW**: Slack community channel: `stackstorm-community.slack.com <https://stackstorm-community.slack.com>`__ (Register `here <https://stackstorm.typeform.com/to/K76GRP>`__.)
```
Comment (manasdk) [inline docs/source/engage.rst]: Please switch to http://stackstorm.com/community-signup

File: docs/source/start.rst [hunk 2]
```diff
@@ -234,10 +240,27 @@ to /opt/stackstorm/packs/, setup, and reload the content:
...
+    # List the key value pairs in the datastore
+    st2 key list
+
+For more information on datastore, check :doc:`datastore`
 
 Troubleshooting
 ----------------------
-If something goes wrong, follow :doc:`/troubleshooting/index` guide, and engage with developers at `#stackstorm on irc.freenode.org <http://webchat.freenode.net/?channels=stackstorm>`__
+If something goes wrong, follow :doc:`/troubleshooting/index` guide, and engage with developers at
+the community channel on slack at `stackstorm-community.slack.com <https://stackstorm-community.slack.com>`__.
```
Comment (dzimine) [inline docs/source/start.rst]: please remove Troubleshooting section from here all together. It's noise.

File: docs/source/engage.rst (hunk 1 above)
Comment (dzimine) [inline docs/source/engage.rst]: better link is `https://stackstorm.com/community-signup`

File: docs/source/start.rst [hunk 3]
```diff
@@ -14,60 +13,70 @@ The best way to explore |st2| is to use CLI. Start by firing a few commands:
...
-operations are also available via REST API, Python, and JavaScript bindings.
-Check the :doc:`CLI and Python Client </reference/cli>` reference for details.
+The default “all-in-one” installation deploys the CLI along with the |st2| services. CLI can be 
+used to access |st2| service remotely. All |st2| operations are also available via REST API, 
+Python, and JavaScript bindings. Check the :doc:`CLI and Python Client </reference/cli>` reference 
+for details.
 
 From v0.8, |st2| ships with WebUI. With the default "all-in-one" installation, you can access it at
-http://hostname:8080/.
+https://hostname_or_ip or http://hostname_or_ip if SSL is not configured.
```
Comment (dzimine) [inline docs/source/start.rst]: we have redirect so http is simpler and will work?

Comment (m4dcoder) [inline docs/source/start.rst]: I already deleted my instance.  But I'll take your word for it.

File: docs/source/start.rst [hunk 4]
```diff
@@ -234,10 +240,27 @@ to /opt/stackstorm/packs/, setup, and reload the content:
 
 For more content - actions, sensors, rules - checkout `st2contrib`_ community repo on GitHub.
 
+Datastore
+---------
+
+While the most data are retrieved as needed by |st2|, you may need to store and share some common
```
Comment (manasdk) [inline docs/source/start.rst]: While the most -. While most

**Comment by manasdk** [comment]
+1

**Comment by dzimine** [comment]
+1

**Pull request merged**